server/src/configureAPI.mjs

```diff
@@ -63,6 +63,9 @@ function configureAPI(app) {
 	// Revert to the way things were when server started up
     app.post('/api/v1/state/revert',                    stateApi.revertState);
 
+	// Start or Stop the interaction log service
+    app.post('/api/v1/state/interactionService',        stateApi.interactionService);
```

Comment:
I'm not sure that this will go in state section of the API. I think I would prefer it in the user section or even on its own. I think we should have two apis. EnableInteractionLogs & DisableInteractionLogs.

app.post('/api/v1/user/interactionService', userApi.enableInteractionLogs);
app.post('/api/v1/user/interactionService', userApi.disableInteractionLogs);

Reply:
added the above changes